karabo_bridge/cli/simulation.py

```diff
@@ -29,6 +29,10 @@ def main(argv=None):
         help='Simulate raw data if True, corrected data if False (default'
              'False)'
     )
+    ap.add_argument(
+        '--reshape', action='store_true',
```

Comment:
I don't like this design: a 'reshape' option implies that whatever happens without it is the normal state, and assumes that people know what that 'normal' state is. It also assumes that there's only one possible reshaping.

Do we need an option at all? I thought the data had a consistent shape, it's just not the same as what the simulator was doing. If we know what the shape should be for a given data source, we shouldn't need an option to control it.

If we do need to control it separately, let's make it properly clear what it means, e.g. `--axis-order pulses,modules,x,y`.

Reply:
The think is that Thomas applied a hotfix in spb to only reshape image.data. So image.data shape of simulator will be different to the shape when connecting to the Pipe2ZMQ device in the SPB topic. Which is very unfortunate. I think this hotfix was a request from the CFEL guys. I am not sure whether or not this should be a long term solution.

Reply:
So there is only one reshape option, either how it comes from the pipeline (no reshape) or reshape then image.data is reshaped to pulses,modules,x,y

Reply:
@takluyver what do you think? Should I leave the reshape parameters, add the one you suggested or simply not include this 'hotpatch'? I tend to say the latter because the bridge is supposed to be agnostic but then again we have the 'hotpatch' in place at SPB. So what you reckon?

Reply:
From discussion on chat: we'll leave it for now; it's easier to add it again later if we need it than to remove it if we don't.

Reply:
Agreed. @antarcticrainforest we should then make sure that the users at SPB are aware of the current difference.